### ISSUE TITLE
Fix `tflint` deprecation warning - "tflint FILE/DIR" is deprecated

### ISSUE THREAD

**Issue by lukasz-mitka** [gruntwork-io/pre-commit#94]
**Describe the solution you'd like**
I don't want to see these warnings
```
WARNING: "tflint FILE/DIR" is deprecated and will error in a future version. Use --chdir or --filter instead.
```

**Describe alternatives you've considered**
Stop using `gruntwork-io/pre-commit`. Don't really want to go there.
Downgrade `tflint`. Don't want to do that either.

**Additional context**
tflint release notes on deprecation - https://github.com/terraform-linters/tflint/releases/tag/v0.46.0
Tested with `gruntwork-io/pre-commit` v0.1.19

**Comment by lukasz-mitka** [comment]
@gcagle3 that could work, but I was hoping for out of the box solution.

**Comment by gcagle3** [comment]
@lukasz-mitka that makes sense, thank you. We'll work to get PR #95 updated with the changes from #92 so we can add this functionality.

**Comment by gcagle3** [comment]
This has been merged and is now available in release [v0.1.22](https://github.com/gruntwork-io/pre-commit/releases/tag/v0.1.22).
